.gitignore

```diff
@@ -7,6 +7,7 @@ models/
 # Local Chroma db
 .chroma/
 db/
+source_documents/
```

Comment:
But we have a `source_documents` by default no? Is the idea to remove the default source too?

Reply:
You do have a point, I was switching branches and that just slipped lol, should change it to directory contents instead of the whole folder.

Reply:
I think if we want to have a default source file, like the state of the union or anything else, it should not be in `.gitignore` either.